dev-support/hbase-personality.sh

```diff
@@ -81,8 +81,11 @@ function personality_globals
 
   # Yetus 0.7.0 enforces limits. Default proclimit is 1000.
   # Up it. See HBASE-19902 for how we arrived at this number.
+  # NOTE: I don't think changing this has an effect. Set the
+  # --proclimit passed to yetus. This seems to do what we
+  # need changing proclimit.
```

Comment:
Ah, yeah. [In yetus](https://github.com/apache/yetus/blob/8b44644/precommit/src/main/shell/test-patch.sh#L96), the env variable is `PROC_LIMIT`. You found a bug :)

Reply:
Let me try filing an issue.